Fix v41 and stable links and update downloads tab (#32)
OWASP/www-project-web-security-testing-guide: index.md

## Stable

> View the always-current stable version at [/stable](stable/).

Reviewer: Should this not be the base url as well? Similar for the latest links

Author: It could use it but theyre static locations so using a relative URL is fine. Whereas with the banner it'll be displayed on all kinds of paths so we need it to link specifically not relatively.

Reviewer: I was thinking more to it being related to local build issues

Author: Did you test it locally and have a problem? It should have been fine. I'll re-test.

Author: Works fine locally, logically should be fine deployed as well.

> View the always-current stable version at [/stable](stable/).

Reviewer 2: Is the link correct? `stable/` and not `/stable`?

Same for `latest` and `v41`.

Author: Yes.

Author: It's relative to index.md, tab_download, etc

I guess we could do it as ./stable/ but that seems unnecessary.

Also the trailing slash is necessary to avoid a redirect, as seen here:
![image](https://user-images.githubusercontent.com/7570458/78505588-74d6c300-7742-11ea-9333-7c869c0a6564.png)

## Latest

We are currently developing release version 5.0.

You can [read the latest development documents in our official GitHub repository](https://github.com/OWASP/wstg/tree/master/document) or view the bleeding-edge content at [/latest](latest/).

## Versioned Releases

> The only currently available versioned releases is [v4.1](v41/).

Reviewer 2: ```suggestion
The only currently available versioned release is [v4.1](v41/).
```

Though this makes it sound like there's never been another. Are we still making the older PDF downloads available on the Releases tab?

Author: I’ll tweak it

## Versioned Releases

> The only currently web hosted versioned releases is [v4.1](v41/). (PDFs of previous releases are still available on the 'Release Versions' tab.)

Reviewer 2: ```suggestion
Only [v4.1](v41/) is currently available as a web-hosted release. Previous releases are available as PDFs on the **Release Versions** tab.
```
